spacy/lang/it/stop_words.py

```diff
@@ -30,21 +30,21 @@
 finche fine fino forse forza fosse fossero fossi fossimo foste fosti fra
 frattempo fu fui fummo fuori furono futuro generale
 
-gia già giacche giorni giorno gli gliela gliele glieli glielo gliene governo
+gia già giacche giorni giorno gli gl' gliela gliele glieli glielo gliene governo
 grande grazie gruppo
 
 ha haha hai hanno ho
 
 ieri il improvviso in inc infatti inoltre insieme intanto intorno invece io
 
-la là lasciato lato lavoro le lei li lo lontano loro lui lungo luogo
+l' la là lasciato lato lavoro le lei li lo lontano loro lui lungo luogo
 
-ma macche magari maggior mai male malgrado malissimo mancanza marche me
+m' ma macche magari maggior mai male malgrado malissimo mancanza marche me
```

Comment:
my Italian is not the strongest, but I wouldn't be sure what is contracted when one write `m'` .

Reply:
It's `mi` . Pronouns such as `si`, `ti` and `mi` are usually shortened, such as this one:  `M'ascolti?`

Reply:
totally true actually :+1: